integration_test_lite.py

```diff
@@ -116,19 +117,22 @@ async def bad_response(self, endpoint):
                 ) as response:
 
                     response_code = response.status
+                    api_info['response_code'] = response_code
+
+                    try:
+                        api_info['response_body'] = await response.json()
+                    except Exception as error:
+                        api_info['response_body'] = {
+                            'raw': await response.text(),
+                            'error': str(error)
+                        }
+
                     if response_code != 200:
-                        api_info['response_code'] = response_code
-
-                        try:
-                            api_info['response_body'] = await response.json()
-                        except Exception as error:
-                            api_info['response_body'] = {
-                                'raw': await response.text(),
-                                'error': str(error)
-                            }
```

Comment:
I think the logic is changed if you moved this out. Is there any reason to do that?

Reply:
You're right. The old logic works just fine